packages/framework/store/src/yjs/awareness.ts

```diff
@@ -109,13 +121,19 @@ export class AwarenessStore<
     } as Flags['readonly']);
   }
 
+  isReadonly$(space: Space) {
+    return computed(() => {
```

Comment:
Instead of generate a computed value every time we call `isReadonly`, can we use a private computed property to track this value?